portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Policies/Policies.tsx

```diff
@@ -302,14 +297,9 @@ const Policies: React.FC = () => {
         oldIndex: number, newIndex: number, target: string, verb: string, currentFlow: string,
     ) => {
         const newApiOperations: any = cloneDeep(apiOperations);
-        let operationInAction = null;
-
-        if (!isChoreoConnectEnabled) {
-            operationInAction = newApiOperations.find((op: any) =>
-                op.target === target && op.verb.toLowerCase() === verb.toLowerCase());
-        } else {
-            operationInAction = newApiOperations.find((op: any) => op.target === target);
-        }
+        let operationInAction = newApiOperations.find((op: any) =>
+            op.target === target && op.verb.toLowerCase() === verb.toLowerCase());
```

Comment:
(Relates to line 224) Can we reuse the `newApiOperations.find((op: any) =>
            op.target === target && op.verb.toLowerCase() === verb.toLowerCase());` ?

Reply:
Since it is only one line, let's keep this for now. We can wait until a duplication of multiple lines occur.